### PR TITLE
Fix a typo in the GitHub.UI.Reactive assembly title

### PR DESCRIPTION
**Pull request by mlaily**

### CODE REVIEW

**Comment by jcansdale** [comment]
@grokys,

> you've submitted PRs recently from other accounts, did you see a similar problem?

Nope, this appears to be new. It's impacting #1667 as well.

File: src/GitHub.UI.Reactive/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using System.Windows;
 using System.Windows.Markup;
 
-[assembly: AssemblyTitle("GitHub.UI.Recative")]
+[assembly: AssemblyTitle("GitHub.UI.Reactive")]
```
Comment (sguthals) [inline src/GitHub.UI.Reactive/Properties/AssemblyInfo.cs]: Thanks for catching this!

**Comment by grokys** [comment]
Merged, thanks @mlaily for spotting this! (and causing us to fix our CI for external contributors in the process ;) )

**Comment by mlaily** [comment]
You are welcome :p

**Pull request merged**